test/data/case3_dc.m

```diff
@@ -0,0 +1,85 @@
+% Case to test adding data to matpower file
```

Comment:
We still have added new DC files, where the br_status of the DC line is set to 1 as opposed to the original test files.